sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -587,6 +591,18 @@ static inline bool gnrc_ipv6_netif_addr_is_non_unicast(const ipv6_addr_t *addr)
  */
 void gnrc_ipv6_netif_init_by_dev(void);
 
+/**
+ * @brief   Get sent and received statistics about IPv6 traffic on this interface.
+ *
+ * @note    This function is only available if compiled with module `netstats_ipv6`.
+ *
+ * @param[in] pid   The PID to the interface.
+ *
+ * @return  A @ref netstats_t pointer to the statistics.
+ * @return  NULL if no statistics are available.
+ */
+netstats_t *gnrc_ipv6_netif_get_stats(kernel_pid_t pid);
```

Comment:
>...and move its "functionality" into the shell.

It depends: Can I still use netstats then if I do not compile the shell, but send the stats over e.g. UDP to a server?